pom.xml

```diff
@@ -2219,9 +2231,6 @@
         </profile>
         <profile>
             <id>jdk16</id>
```

Comment:
Do we still need this profile `jdk16 ` ?

Reply:
Nope (we could remove `jdk17` too) but that would require some further cleanup (CircleCI, etc.). Would rather punt it for now but I can come back to it.